contrib/inventory/brook.py

```diff
@@ -0,0 +1,262 @@
+#!/usr/bin/env python
+# Copyright 2016 Doalitic.
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+"""
+Brook.io external inventory script
+==================================
+
+Generates inventory that Ansible can understand by making API requests to Brook.io via the libbrook
+library. Hence, such dependency must be installed in the system to run this script.
+
+The default configuration file is named 'brook.ini' and is located alongside this script. You can
+choose any other file by setting the BROOK_INI_PATH environment variable.
```

Comment:
Don't have a strong preference, but `BROOK_INI_PATH` seems to be consistent with other dynamic inventories like apache-libcloud, ec2, and gce.